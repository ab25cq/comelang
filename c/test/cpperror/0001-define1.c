/*
PATTERN: redefinition
PATTERN: c:7:
*/

#define X Y
#define X Z


