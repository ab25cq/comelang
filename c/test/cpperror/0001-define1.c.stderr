redefinition of macro X at test/cpperror/0001-define1.c:7:9
#define X Z
        ^
